checkstyle/checkstyle.xml

```diff
@@ -71,9 +71,11 @@
     </module>
     <module name="PackageName"/>
     <module name="ParameterName"/>
+    <!--
```

Comment:
I have commented this *temporarily* since a non-final static reference was used in `FlinkPravegaInputFormatITCase` code.

Reply:
Use a targeted suppression if possible.
https://stackoverflow.com/questions/27688426/ignoring-of-checkstyle-warnings-with-annotation-suppresswarnings